docs/core/compatibility/windows-forms/6.0/listview-invalidoperationexception.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Breaking change: ListViewGroupCollection methods throw new InvalidOperationException"
+description: Learn about the breaking change in .NET 6 where some ListViewGroupCollection methods throw a new InvalidOperationException if the ListView is in virtual mode.
+ms.date: 09/23/2021
+---
+# ListViewGroupCollection methods throw new InvalidOperationException
+
+Previously, an <xref:System.InvalidOperationException> was thrown if <xref:System.Windows.Forms.ListViewGroupCollection> methods were invoked on a <xref:System.Windows.Forms.ListViewItem.ListView> in virtual mode *and* the <xref:System.Windows.Forms.Control.Handle> had already been created. Starting in .NET 6, these <xref:System.Windows.Forms.ListViewGroupCollection> methods now only check if the <xref:System.Windows.Forms.ListViewItem.ListView> is in virtual mode. If it is, they throw an <xref:System.InvalidOperationException> with a more descriptive message.
```

Comment:
Is `<xref:System.Windows.Forms.Control.Handle>` the right type of handle? There were a few to choose from.

Reply:
Yes that is.